src/Aquifer.Public.API/Endpoints/Bibles/Books/List/Endpoint.cs

```diff
@@ -2,13 +2,14 @@
 using Aquifer.Public.API.Helpers;
 using FastEndpoints;
 
-namespace Aquifer.Public.API.Endpoints.BibleBooks.List;
+namespace Aquifer.Public.API.Endpoints.Bibles.Books.List;
 
 public class Endpoint : EndpointWithoutRequest<IEnumerable<Response>>
 {
     public override void Configure()
     {
-        Get("/bible-books");
+        // "/bible-books" is still supported for legacy compatibility but "/bibles/books" is the new hotness
```

Comment:
I think we can remove these comments since the LegacyEndpoint implies it.

Reply:
Done.